changelog.d/16013.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Synapse 1.89.0 where the `redacts` property was not properly added to the `content` for forwards-compatibility with room versions 1 through 10.
```

Comment:
This isn't really a bugfix -- the forwards compatible is a "should" in the MSC, but I don't know where else to put this.

```suggestion
Fix a bug introduced in Synapse 1.89.0rc1 where the `redacts` property was not properly added to the `content` for forwards-compatibility with room versions 1 through 10.
```